Code/Sideloaded-TextTracks-Examples/Cocoapod/SideloadedTextTracksExample/ViewController.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  ViewController.swift
+//  SideloadedTextTracksExample
+//
+//  Created by Wonne Joosen on 13/04/2024.
+//
+
+import UIKit
+import THEOplayerSDK
+import THEOplayerConnectorSideloadedSubtitle
+
+class ViewController: UIViewController {
+    
+    
+    @IBOutlet weak var playerViewContainer: UIView!
+    var player: THEOplayer
+    
+    required init?(coder: NSCoder) {
+//        self.player = THEOplayer(with: nil, configuration: THEOplayerConfiguration(chromeless: false, ads: AdsConfiguration(
+//            showCountdown: true,
+//            preload: AdPreloadType.MIDROLL_AND_POSTROLL,
+//            googleIma: GoogleIMAAdsConfiguration(useNativeIma: false)
+//        )))
+        let playerConfiguration = THEOplayerConfigurationBuilder()
+        playerConfiguration.license = "<your license>"
```

Comment:
Better to keep this line commented out. When running the example app we run into a license error because of it. The sources specified in the app run just fine without the license key.

Reply:
✅